src/hubot-chatops.coffee

```diff
@@ -21,7 +21,12 @@ module.exports = (robot) ->
   deploy = (repoName, branchName, res) ->
     room = if robot.adapterName == 'hipchat' then res.message.user.reply_to else res.message.room
 
-    repo = "https://github.com/#{repoName}.git"
+    if (repoName.match(/.*[:\/](.*)\/(.*).git/)){
+      repo = repoName
+    } else {
+      repo = "https://github.com/#{repoName}.git"
+    }
```

Comment:
I'm not able to see the code change which you are mentioning . Has the code been pushed ?